### ISSUE TITLE
February-March 2022 PR Sprint

### ISSUE THREAD

**Issue by Hans5958** [ScratchAddons/website-v2#161]
This issue lists all the PRs that is on hold due to i18n changes. It is to be merged all together if possible.

(checked means ready to be merged)

- [ ] #149 (uncertain)
- [x] #151 (needs minor edit after #153)
	- [x] related: ScratchAddons/contributors#196
- [x] #153
- [x] #154 
- [x] #155
- [x] #156
- [x] #158
- [x] #159
- [x] #160
	- [x] related: ScratchAddons/ScratchAddons#4335
- [x] #165 (needs minor edit after #153)

**Comment by Hans5958** [comment]
Translation Memory Fillup has been activated on Transifex (website only) to prevent retranslation when it is merged.

**Comment by Hans5958** [comment]
Just added #165.

**Comment by Hans5958** [comment]
I think all my PRs are ready to be merged.

@WorldLanguages, waiting for you with the WIP, or a green light for those that are ready to be merged.

**Comment by Hans5958** [comment]
This PR is due to be edited later due to i18n key change from #159.

- #151
- #165

**Comment by Hans5958** [comment]
Placeholders have been added on necessary files. Now it should be easier than ever to create content.

**Comment by Hans5958** [comment]
Alright, I'm going to start merging PRs right now,

**Comment by Hans5958** [comment]
All PRs have been merged (and all the necessary edits have been made), so it's time to close this PR. Thank you so much for those who have helped on this sprint.
